Source/WebKit/GPUProcess/graphics/WebGPU/RemoteGPU.h

```diff
@@ -63,10 +63,12 @@ class ObjectHeap;
 struct RequestAdapterOptions;
 }
 
+typedef Function<void(WebCore::MediaPlayerIdentifier, Function<void(WebCore::MediaPlayer&)>&&)> PerformWithMediaPlayerOnMainThread;
```

Comment:
I think we generally prefer `using PerformWithMediaPlayerOnMainThread = Function<void(WebCore::MediaPlayerIdentifier, Function<void(WebCore::MediaPlayer&)>&&)>` rather than `typedef`

Also, can we break this into two lines?
```
using MediaPlayerAccessor = Function<void(WebCore::MediaPlayer&)>;
using PerformWithMediaPlayerOnMainThread = Function<void(WebCore::MediaPlayerIdentifier, MediaPlayerAccessor&&)>;
```